src/main/java/com/epam/aidial/core/data/ResourceUrl.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.aidial.core.data;
+
+import com.epam.aidial.core.storage.BlobStorageUtil;
+import com.epam.aidial.core.util.UrlUtil;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class ResourceUrl {
```

Comment:
what's the value of this class? Can you use ResourceDescription instead?

Reply:
I need a more generic url without enforcing resource type and bucket. Because I do not have them in some places.